src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const newState = { ...state };
+  const returnObjects = [];
+  let container;
+
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i].type) {
+      case 'clear':
+        for (const key in newState) {
+          delete newState[key];
+        }
+        container = { ...newState };
```

Comment:
you don't need container, push copy of newState to the returnObjects at the end of the loop